ydb/public/lib/ydb_cli/commands/ydb_admin.cpp

```diff
@@ -10,6 +11,7 @@ TCommandAdmin::TCommandAdmin()
 {
     AddCommand(std::make_unique<NDynamicConfig::TCommandConfig>());
     AddCommand(std::make_unique<NDynamicConfig::TCommandVolatileConfig>());
+    AddCommand(std::make_unique<NCluster::TCommandCluster>());
```

Comment:
Aren't config and volatile-config cluster-wide?
Was there a support meeting about this?

Reply:
it's still a draft

Reply:
We have discussed this, and decided to use "admin cluster bootstrap" command to emphasize the fact that this applies to the whole cluster.